src/renderer/init.js

```diff
@@ -84,7 +102,14 @@ async function init() {
     await hardReset();
   }
 
-  const store = createStore({ dbMiddleware });
+  const store = createStore({
+    dbMiddleware,
+    analyticsMiddleware: analytics({
+      migration: {
+        ...(oldUserId ? { oldUserId } : {}),
+      },
```

Comment:
probably can just be `migration: oldUserId ? { oldUserId } : null`

Reply:
yes, i was thinking if we use this migration thing with other stuff in the futur

Reply:
but we don't so i would not anticipate something we don't need now.